emails/migrations/0051_abusemetrics_num_email_forwarded_per_day.py

```diff
@@ -0,0 +1,82 @@
+# Generated by Django 3.2.14 on 2022-08-29 20:47
+
+from django.db import migrations, models
+
+
+def add_db_default_forward_func(apps, schema_editor):
+    """
+    Add field num_email_forwarded_per_day to abusemetrics
+
+    Using `./manage.py sqlmigrate` for the SQL, and the technique from:
+    https://stackoverflow.com/a/45232678/10612
+    """
+    if schema_editor.connection.vendor.startswith("postgres"):
+        schema_editor.execute(
+            'ALTER TABLE "emails_abusemetrics"'
+            ' ALTER COLUMN "num_email_forwarded_per_day" SET DEFAULT 0;'
+        )
+    elif schema_editor.connection.vendor.startswith("sqlite"):
+        schema_editor.execute(
+            'CREATE TABLE "new__emails_abusemetrics"'
+            ' ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT,'
+            ' "num_email_forwarded_per_day" smallint unsigned NOT NULL CHECK ("num_email_forwarded_per_day" >= 0),'
```

Comment:
This needs a default:

```suggestion
            ' "num_email_forwarded_per_day" smallint unsigned NOT NULL DEFAULT 0 CHECK ("num_email_forwarded_per_day" >= 0),'
```